### PR TITLE
fix: MultiAction.add() return

### LINKED ISSUE #956: MultiAction.add() Incorrect Docstring
## The problem
In appium/webdriver/common/multi_action.py:

MultiAction.add method docstring states that it returns self instance. The method's type hint states that it returns none.

````    
    def add(self, *touch_actions: 'TouchAction') -> None:
        """Add TouchAction objects to the MultiAction, to be performed later.

        Args:
            touch_actions: one or more TouchAction objects describing a chain of actions to be performed by one finger

        Usage:
            | a1 = TouchAction(driver)
            | a1.press(el1).move_to(el2).release()
            | a2 = TouchAction(driver)
            | a2.press(el2).move_to(el1).release()
            | MultiAction(driver).add(a1, a2)

        Returns:
            `MultiAction`: Self instance
        """
        for touch_action in touch_actions:
            if self._touch_actions is None:
                self._touch_actions = []

            self._touch_actions.append(copy.copy(touch_action))
````

## Environment
- Appium version (or git revision) that exhibits the issue: 
    - Currently on master branch, 3.3.1
- Last Appium version that did not exhibit the issue (if applicable):
    - docstring was updated in this commit: https://github.com/appium/python-client/commit/8205776240b4c92e021064751e5b113d0c915d50#diff-a825035d2eb075516862e3f675dee371524899335873de35074e9300d63b0d9f
    - type hint was added in this commit: https://github.com/appium/python-client/commit/9c597a3afe76ff1283c45259ed6e1c4aadac530d#diff-a825035d2eb075516862e3f675dee371524899335873de35074e9300d63b0d9f

### PR DESCRIPTION
**Pull request by mmonfared**
closes #956

### CODE REVIEW

**Review by mykola-mokhnach — APPROVED** [review]

File: appium/webdriver/common/multi_action.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def __init__(self, driver: 'WebDriver', element: Optional['WebElement'] = None)
         self._element = element
         self._touch_actions: List['TouchAction'] = []
 
-    def add(self, *touch_actions: 'TouchAction') -> None:
+    def add(self, *touch_actions: 'TouchAction') -> 'MultiAction':
```
Comment (mykola-mokhnach) [inline appium/webdriver/common/multi_action.py:48]: where does it return?

Comment (mmonfared) [reply appium/webdriver/common/multi_action.py]: Updated. Just docstring return value updated now to align with the return value (None). Please check.

Comment (mykola-mokhnach) [reply appium/webdriver/common/multi_action.py]: we plan anyway to [remove](https://github.com/appium/python-client/pull/960) these endpoints soon, but for the sake of correctness lets have this patch merged

Comment (mmonfared) [reply appium/webdriver/common/multi_action.py]: Yes, I'm aware of that. After seeing it in the issues, I thought why not correct it for now.

**Comment by KazuCocoa** [comment]
Maybe proper fix is adding `return self` at the end (according to the documentation in https://github.com/appium/python-client/blob/6af9c8abbb8aa2552cedda8046a3d91903a53b70/appium/webdriver/common/multi_action.py#L78 )

**Comment by KazuCocoa** [comment]
Thank you, I made https://github.com/appium/python-client/pull/964 to return self. Maybe this is a proper correction

**Pull request closed without merging**